README.md

```diff
@@ -167,6 +171,24 @@ https://www.python.org/downloads/
 
 go get -u github.com/jteeuwen/go-bindata/...
 
+## Elasticsearch indexation
+
+	Detect local direction for [mdb] section in config.toml (url="postgres://localhost/mdb?sslmode=disable&user=postgres&password=???")
+	
+	Detect local direction for [elasticsearch] section in config.toml (url="http://127.0.0.1:9200")
+	
+	Open Terminal environment and in command line change directory to archive-backend project's folder, type:
+		```Shell
+		go build
+
+	In command line type (indexation will take some time , something like 40 minutes ):
+		```Shell
+		archive-backend index
+		
+	In command line type:
+		```Shell
+		archive-backend index_grammars
+
```

Comment:
Fix formatting in 'Elasticsearch indexation' section. This is how it looks like now: https://github.com/Bnei-Baruch/archive-backend/tree/Modify_Readme_ExplainIndexGrammar#elasticsearch-indexation